<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="nomeLabel">
          <property name="text">
           <string>Nome</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nomeLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="enderecoLabel">
          <property name="text">
           <string>Endereco</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="enderecoLineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="idadeLabel">
          <property name="text">
           <string>Idade</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="idadeSpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_verifica">
            <property name="text">
             <string>Verifica</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_submete">
            <property name="text">
             <string>Submeter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>325</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_verifica</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>RecebeCliqueVerifica()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_submete</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>RecebeCliqueSubmeter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>HabilitaSubmeter(bool)</signal>
   <receiver>pushButton_submete</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>HabilitaSubmeter(bool)</signal>
  <slot>RecebeCliqueVerifica()</slot>
  <slot>RecebeCliqueSubmeter()</slot>
 </slots>
</ui>
